ServiceDesk.Abstractions/ArcadiaTeamsBot.ServiceDesk.Abstractions(Delete).csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's this?

Reply:
Folders left after renaming `ServiceDesk `and `ServiceDesk.Abstraction` projects.
Already fixed.